include/net/net_app.h

```diff
@@ -387,18 +387,21 @@ struct net_app_ctx {
 		} mbedtls;
 
 		/** Have we called connect cb yet? */
-		bool connect_cb_called;
+		u8_t connect_cb_called : 1;
```

Comment:
These  type changes for existing fields are not mentioned in the patch description.

Reply:
True, I will add comment about these to description.